package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nahmii-sdk",
-  "version": "2.5.1",
+  "version": "2.6.0",
```

Comment:
The changes to monetaryamount are not backwards compatible so this should be a major bump

Reply:
I have bumped the major version now. 

I was originally in doubt whether to bump it, since we normally don't use the constructor directly.